docs/content/stepbystep/Git-setup.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Git setup'
+metaTitle: 'Installing and using git | Discordeno'
+metaDescription: 'Setting up git for your pc'
+---
+
+# Installing
+
+First, go to the [Git download page](https://git-scm.com/downloads) and download the version for your OS. I'm using windows so I'll use <https://git-scm.com/download/win>
+
+Once downloaded click next and next again (unless you need to change the install location)
+
+Make sure that on the select component screen all these are selected:
+
+- Git LFS
+- Associate .git files
+
+Now click next, If you have a start menu folder screen just click next.
+
+Now it will ask you for your default editor. For now set this to custom and make it `notepad`
```

Comment:
notepad not VSC?

Reply:
@Skillz4Killz I would just recommend the default terminal.

Reply:
Whatever you think best, tbh i think it would be easier to keep beginners on VSC terminal but its ur call